TCAT/Cells/LargeDetailTableViewCell.swift

```diff
@@ -55,77 +55,77 @@ class LargeDetailTableViewCell: UITableViewCell {
         detailLabel.sizeToFit()
         return detailLabel
     }
-    
+
     override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
-        
+
         chevron = getChevron()
         contentView.addSubview(chevron)
-        
+
         titleLabel = getTitleLabel()
         contentView.addSubview(titleLabel)
-        
+
         detailLabel = getDetailLabel()
         contentView.addSubview(detailLabel)
-        
+
     }
-    
+
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
+
     override func prepareForReuse() {
-        
+
 //        busIconView.prepareForReuse()
```

Comment:
delete commented out code if unnecessary

Reply:
This is outside the scope of this PR. Will be part of larger refactoring.